src/pages/ProgramsList/ProgramItems.js

```diff
@@ -26,6 +30,10 @@ for (let i = 0; i < 23; i++) {
   dummyData.push({
     id: i,
     title: `Survey ${i}`,
+    avatar: "https://image.flaticon.com/icons/svg/3163/3163231.svg", //depends on type of program
```

Comment:
nit: comment can be in a separate line above
`avatar: "...."`
to keep it 80 characters per line